src/test/xmeta/test_xmeta_emit.cpp

```diff
@@ -350,23 +387,30 @@ void VerifyTypeRefName(std::string const& expectedName, std::string_view current
     REQUIRE(combineName == expectedName);
 }
 
-void VerifyReturnType(ElementType const& expectedType, std::string const& expectedTypeRef, MethodDef const& method, TypeDef const& parent)
+using value_type = std::variant<ElementType, coded_index<TypeDefOrRef>, GenericTypeIndex, GenericTypeInstSig, GenericMethodTypeIndex>;
```

Comment:
This is a duplicate of `TypeSig::value_type`. Instead of duplicating it here, either use it directly, or define your local type in terms of it, aka `using value_type = TypeSig::value_type`.